pages/sigmaker.tsx

```diff
@@ -1,9 +1,109 @@
 import type { NextPage } from "next";
-import { ThemeProvider } from "@mui/material";
+import {
+  Button,
+  Container,
+  Divider,
+  FormControl,
+  Grid,
+  Stack,
+  TextField,
+  ThemeProvider,
+  Typography,
+} from "@mui/material";
 import { theme } from "../styles/theme";
+import { useTheme } from "@mui/material/styles";
+import { ChangeEvent, useState } from "react";
+import useMediaQuery from "@mui/material/useMediaQuery";
 
 const Sigmaker: NextPage = () => {
-  return <ThemeProvider theme={theme}></ThemeProvider>;
+  const themeMUI = useTheme();
+  const isNotMobile = useMediaQuery(themeMUI.breakpoints.up("sm"));
+
+  const [formData, setFormData] = useState({
+    fullName: "",
+    title: "",
+    phone: "",
+    email: "",
+  });
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    setFormData({
+      ...formData,
+      [name]: value,
+    });
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Container sx={{ mt: 4 }}>
```

Comment:
do we have a convention for using inline styles or using stylesheets? nothing wrong with this, just wondering for the future

Reply:
We can mainly use MUI styling but if we need we can add the css in the corresponding file!! 

Also I ended up not adding an sigmaker.css in this PR since I realized it'd be empty but we can add it once we need css! Maybe in your + dhivas' PR or the future!